Bottom: f16f9254eca01a54d5524258d61a6db35f2ce449
Top:    0c39fca4f899110ca8fb93b8b89c629752af41b8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 16:18:52 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
